packages/lesswrong/components/common/RecommendedReading.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { Components, registerComponent } from 'meteor/vulcan:core';
+import { Link } from 'react-router';
+import withUser from '../common/withUser';
+import { withStyles } from '@material-ui/core/styles';
+import Hidden from '@material-ui/core/Hidden';
+
+const styles = theme => ({
+  smallScreenRecommendedReading: {
+    maxWidth: 720,
+    margin: "0 auto",
+  },
+});
+
+
+const RecommendedReading = ({currentUser, classes}) => {
+  const sectionTitle = currentUser ? "Recommended Sequences" : "Recommended Reading";
+  
+  const sectionWrapper = (recommendedContent) => (
+    <Components.Section
+      title={sectionTitle}
+      titleLink="/library"
+      titleComponent= {<Components.SectionSubtitle to="/library">
+        <Link to="/library">Sequence Library</Link>
+      </Components.SectionSubtitle>}
+    >{recommendedContent}</Components.Section>
+  );
+  
+  const omittedSectionWrapper = (content) => (
```

Comment:
Design wise, I think not having any section title on phones is quite confusing. As a logged in user I don't know what the top thing is supposed to be, and why it's shown to me. In addition, it also makes it much harder to find more recommended sequences, or recommended reading, since I can no longer click on the section title to go the library section. 

I think omitting the titles isn't worth it. It saves less than 50 pixels for quite little gain and I expect will overall make parsing the page a lot harder.

Reply:
Actually, it's 90 pixels. Logged out on the smallest screen size, it makes the difference between 1.5 posts being visible, and 0 posts being visible.

Reply:
Maybe we can scrounge up enough pixels to afford a header here. I'd start with editing the copy of the R:A-Z description, since it's not very good and also currently takes up 7 lines. Next thing to go would probably be the padding between the map graphic and the Curated Posts section, followed by the map graphic itself.

But, there's not really a free lunch here; mobile front-page above-the-fold has *very* little space.

Reply:
Yeah, I think we should really try to keep the header here. Editing the R:A-Z description seems good, agree that it's not that great atm.

Reply:
But yeah, I still think for logged-out users, we just can't show posts above the fold, and shouldn't try to. For logged-in users I think we already have at least one or two posts above the fold even with the header, as long as we remove the second two recommended sequences, which I am in favor of.